src/cljs/main/broadfcui/page/profile.cljs

```diff
@@ -26,6 +26,14 @@
         (let [loc (.-location js/window)]
           (str (.-protocol loc) "//" (.-host loc) "/#profile/nih-username-token={token}")))))
 
+(defn get-fence-link-href []
+  (str "https://qa.dcf.planx-pla.net" #_(get @config/config "fenceUrlRoot")
+       "/user/oauth2/authorize?response_type=code&client_id=REPLACE_ME&redirect_uri="
```

Comment:
you might consider making `REPLACE_ME` its own string, to make it stand out even more